Automattic/wp-calypso
# Jetpack Backup: Move download ready icon to assets folder and import it as a module

## Proposed Changes

* Move download ready icon to the assets folder and import it as a module

## Testing Instructions



* Checkout this branch locally and navigate to the backup download page (pick a backup and then click on `Download backup`)
* Ensure the download ready icon in the header loads properly

## Pre-merge Checklist
